pages/api/augment-request.ts

```diff
@@ -1,7 +1,14 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
```

Comment:
We can't avoid breaking these rules here because of the limitations of the types available to us.